src/components/Tokens/TokenTable/SearchBar.tsx

```diff
@@ -16,14 +16,14 @@ const SearchInput = styled.input`
   background-image: url(${searchIcon});
   background-size: 20px 20px;
   background-position: 12px center;
-  background-color: 'transparent';
+  background-color: ${({ theme }) => theme.backgroundSurface};
```

Comment:
did design say to do this? https://www.figma.com/file/kNSDBMpOzxSTOP6MerohLm/Web-App-Spec-(Phase-0-%26-Phase-1)-%5BWIP%5D?node-id=1708%3A26046 

looks like it's transparent?

Reply:
<img width="685" alt="Screen Shot 2022-08-22 at 6 09 36 PM" src="https://user-images.githubusercontent.com/4899429/186027433-19afc9e5-e4b0-4d99-b6ca-a6bdfa31c9c7.png">